### PR TITLE
Add data for the CSSAnimations interface.

### PR DESCRIPTION
**Pull request by jpmedley**
The runtime flag gating this was flipped from experimental to stable in 84. https://storage.googleapis.com/chromium-find-releases-static/9cd.html#9cdcb106b0bcfee75cf3e94576fd48c055cc4bd6

The file to look at is [runtime_enabled_features.json5](https://chromium.googlesource.com/chromium/src/+/9cdcb106b0bcfee75cf3e94576fd48c055cc4bd6%5E%21/#F0).

### CODE REVIEW

**Comment by ddbeck** [comment]
Is there a spec for this? This interface is not named in the CSS Animations spec or the Web Animations API spec and I wasn't sure where else to look.

**Comment by jpmedley** [comment]
https://drafts.csswg.org/css-animations-2/#the-CSSAnimation-interface

File: api/CSSAnimation.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": "84"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (ddbeck) [inline api/CSSAnimation.json:93]:
```suggestion
            "standard_track": true,
```

File: api/CSSAnimation.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": true
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply api/CSSAnimation.json]: I don't have any information, so I changed it to null.

File: api/CSSAnimation.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            "version_added": null
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": true
```
Comment (ddbeck) [inline api/CSSAnimation.json:38]:
```suggestion
            "version_added": null
```

File: api/CSSAnimation.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+              "version_added": null
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": true
```
Comment (ddbeck) [inline api/CSSAnimation.json:85]:
```suggestion
              "version_added": null
```

**Pull request merged**